### PR TITLE
use pytest_randomly

### PR DESCRIPTION
**Pull request by joamatab**
- speed up tests

### CODE REVIEW

**Comment by MatthewMckee4** [comment]
Would be good to add pytest xdist to requirements too

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ test: test-data-gds
 	pytest -s
 
 test-force:
-	run pytest --force-regen -s
+	pytest -n auto --randomly-seed=41 --force-regen
```
Comment (tvt173) [inline Makefile:34]:
```suggestion
	pytest -n auto --force-regen
```

why seed? doesn't that defeat the purpose?

File: pyproject.toml [hunk 2]
```diff
@@ -90,7 +90,8 @@ maintainer = [
   "mypy",
   "tbump",
   "autotyping",
-  "towncrier"
+  "towncrier",
+  "pytest-randomly"
```
Comment (tvt173) [inline pyproject.toml:94]: do we not need pytest-xdist?

**Review by tvt173 — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (sebastian-goeldi) [reply pyproject.toml:94]: also, I believe we can remove towncrier now?

File: Makefile (hunk 1 above)
Comment (joamatab) [reply Makefile]: Good, catch! i fixed it

File: pyproject.toml (hunk 2 above)
Comment (joamatab) [reply pyproject.toml]: we still use towncrier to add the dates on the changelog

**Pull request merged**